src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -121,6 +121,20 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as Theme[])('%s', theme
     })
   );
 
+  test(
+    'preserves tools inner content state when drawer open and close',
```

Comment:
btw, do you see much value in this integ test? you've already covered this case in the unit test below, so they are basically duplicate each other